### PR TITLE
DLP: Added sample for inspect storage with SCC integration

### PR DESCRIPTION
**Pull request by jainamtrivedi-crest**
- Added sample for [Inspect Storage with SCC Integration](https://cloud.google.com/dlp/docs/sending-results-to-scc.md#cloud-storage-data) and test cases

### CODE REVIEW

File: dlp/api/Snippets/InspectStorageWithSCCIntegration.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+public class InspectStorageWithSCCIntegration
+{
+    public static DlpJob SendGcsData(
+        string projectId,
+        string gcsPath,
+        Likelihood minLikelihood = Likelihood.Unlikely,
+        IEnumerable<InfoType> infoTypes = null)
+    {
+        // TODO
```
Comment (annadata) [inline dlp/api/Snippets/InspectStorageWithSCCIntegration.cs:31]: Clean up required?

File: dlp/api/Snippets/InspectStorageWithSCCIntegration.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        // Instantiate the dlp client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct the storage config by specifying the file to be inspected.
```
Comment (annadata) [inline dlp/api/Snippets/InspectStorageWithSCCIntegration.cs:35]: Please align comments with [Java snippet](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/fafc6dca9dad437b9de9062d8f54168396fb0109/dlp/snippets/src/main/java/dlp/snippets/InspectGcsFileSendToScc.java#L62).

File: dlp/api/Snippets/InspectStorageWithSCCIntegration.cs [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+        // Specify the type of info to be inspected and construct the inspect config.
+        var inspectConfig = new InspectConfig
+        {
+            InfoTypes =
+            {
+                infoTypes ?? new InfoType[]
+                {
+                    new InfoType { Name = "EMAIL_ADDRESS" },
+                    new InfoType { Name = "PERSON_NAME" }
```
Comment (annadata) [inline dlp/api/Snippets/InspectStorageWithSCCIntegration.cs:55]: Please align infotypes with [java snippet](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/fafc6dca9dad437b9de9062d8f54168396fb0109/dlp/snippets/src/main/java/dlp/snippets/InspectGcsFileSendToScc.java).

**Review by annadata — COMMENTED** [review]
Please compare this sample with protocol/java sample.

File: dlp/api/Snippets/InspectStorageWithSCCIntegration.cs (hunk 1 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/InspectStorageWithSCCIntegration.cs]: Sorry my bad. Forgot to remove comment. Thanks.

File: dlp/api/Snippets/InspectStorageWithSCCIntegration.cs (hunk 2 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/InspectStorageWithSCCIntegration.cs]: Done.

File: dlp/api/Snippets/InspectStorageWithSCCIntegration.cs (hunk 3 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/InspectStorageWithSCCIntegration.cs]: Done.

**Comment by jainamtrivedi-crest** [comment]
> Please compare this sample with protocol/java sample.

Done. Thanks.

**Review by annadata — APPROVED** [review]

**Review by amanda-tarafa — APPROVED** [review]

**Pull request merged**